src/data/__tests__/cooking_times.test.js

```diff
@@ -21,5 +21,47 @@ import {
 describe('this test prevents to any issues and problems, also to break the structure of cooking_times data', () => {
   test('cooking_times data files returns object', () => {
     expect(cooking_times).not.toBe('')
```

Comment:
change cooking_times on one variable from imports. instead of cooking_times use frenchBeans. because cooking_times doesn't exist, it's just file. also change in other subtests on variable from import